people/templates/people/teams.html

```diff
@@ -0,0 +1,19 @@
+{% extends "people/base.html" %}
+{% load menu_generator %}
+
+{% block title %}Teams{% endblock %}
+
+{% block content %}
+
+{% for team in teams %}
+    <div class="people-teams-team">
+        <h2><a href="{% url 'people:team' team.slug %}">{{ team }}</a></h2>
+        <p><small>{{ team.person_set.all|length }} member(s) in this team:
+        {% for person in team.person_set.all %}
+            <a href="{% url 'root:person' person.login %}">{{ person.login }}</a>
+        {% endfor %}
+    </small></p>
```

Comment:
This indentation is a bit odd.

```suggestion
        <p>
            <small>
                {{ team.person_set.all|length }} member(s) in this team:
                {% for person in team.person_set.all %}
                <a href="{% url 'root:person' person.login %}">{{ person.login }}</a>
                {% endfor %}
            </small>
        </p>
```

Does it look well with a team with several people?

Reply:
It does, you can see the same snippet working at the "Interest vs. knowledge" view at our production inststance.